e2e-tests/IIoTPlatform-E2E-Tests/Standalone/A_PublishSingleNodeStandaloneDirectMethodTestTheory.cs

```diff
@@ -79,16 +91,15 @@ IIoTMultipleNodesTestContext context
             ).GetAwaiter().GetResult());
             Assert.Null(exception);
 
-            //Todo
-            //Here instantiate _directMethod class and call the Publish direct method
-            //This is just a sample call
-            //_directMethods = new DirectMethodsAPIClient(_iotHubConnectionString, _iotHubPublisherDeviceName, _iotHubPublisherModuleName);
+            var request = model.ToApiModel();
```

Comment:
I think we should not apply this transformation, the payload needs to contain the api model, not the internal model.